File: src/main/java/archives/tater/keepinvfilter/KeepInvFilterTags.java
package archives.tater.keepinvfilter;

import net.minecraft.item.Item;
import net.minecraft.registry.RegistryKeys;
import net.minecraft.registry.tag.TagKey;
import net.minecraft.util.Identifier;

public class KeepInvFilterTags {
    public static TagKey<Item> KEEP_ITEMS = TagKey.of(RegistryKeys.ITEM, new Identifier(KeepInvFilter.MOD_ID, "keep"));
}

File: src/main/kotlin/archives/tater/keepinvfilter/KeepInvFilterTags.kt
package archives.tater.keepinvfilter

import net.minecraft.item.Item
import net.minecraft.registry.RegistryKeys
import net.minecraft.registry.tag.TagKey
import net.minecraft.util.Identifier

object KeepInvFilterTags {
    @JvmField
    val KEEP_ITEMS: TagKey<Item> = TagKey.of(RegistryKeys.ITEM, Identifier(KeepInvFilter.modId, "keep"))
}
